unbxdAutosuggest.js

```diff
@@ -1803,10 +1803,19 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, params) {
 				+ '{{/if}}';
 		}
 		, prepareHTML: function () {
-			var html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall">',
-				self = this,
-				mainlen = 0,
-				sidelen = 0;
+			var html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall">';
+
+			if (this.options.isMobile) {
+				if (this.options.isMobile()) {
+					html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall mobile">';
+				}
+			} else if (isMobile.any()) {
+				html = '<ul class="unbxd-as-maincontent unbxd-as-suggestions-overall mobile">';
+			}
+			
+			var self = this,
+			mainlen = 0,
```

Comment:
please add comment on resetting these variables.

Reply:
I guess this is not variable resetting but just a variable declaration that was already present but just moved to a new line, so must be okay

Reply:
No variables have been reset. it is moved to another line due to additional code.